src/logcollector/logcollector.c

```diff
@@ -682,12 +682,12 @@ void LogCollectorStart()
                         current->exists = 1;
                         char msg_alert[512 + 1];
 
-                        snprintf(msg_alert, 512, "ossec: File size reduced "
+                        snprintf(msg_alert, 512, "wazuh: File size reduced "
                                  "(inode remained): '%s'.",
                                  current->file);
 
                         /* Send message about log rotated */
-                        w_msg_hash_queues_push(msg_alert, "wazuh-logcollector", strlen(msg_alert) + 1, default_target, LOCALFILE_MQ);
+                        w_msg_hash_queues_push(msg_alert, "ossec-logcollector", strlen(msg_alert) + 1, default_target, LOCALFILE_MQ);
```

Comment:
Here, we are replacing "**wazuh**-logcollector" by "**ossec**-logcollector"

Reply:
It's true! This was a rebase error. Nice catch.